tests/unit/test_genotype_phenotype.py

```diff
@@ -187,4 +187,76 @@ def testExtractAssociationsDetails(self):
         pass
 
     def testToGA4GH(self):
-        pass
+        sample_associations = {
```

Comment:
per your conversation - if we start with a query it is not really a unit test of toGA4GH.  That being said, it might make it more understandable if we added a comment here that describes this as the intermediate result of a query ...